rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -517,6 +518,7 @@ namespace vk
 			barrier.srcAccessMask = VK_ACCESS_SHADER_READ_BIT | VK_ACCESS_INPUT_ATTACHMENT_READ_BIT;
 			src_stage = VK_PIPELINE_STAGE_FRAGMENT_SHADER_BIT;
 			break;
+		default: break;
```

Comment:
default should throw "Attempted to transition from invalid layout"

Reply:
Making this change causes Vulkan to crash the emulator in all games, as discovered by our favorite Finn Jotain. Falling through allows normal behavior.